test/fixture/config/agent_good_windows.conf

```diff
@@ -0,0 +1,42 @@
+agent {
+    bind_address = "127.0.0.1"
+    bind_port = "8088"
+    data_dir = "."
+    log_level = "INFO"
+    server_address = "127.0.0.1"
+    server_port = "8081"
+    trust_bundle_path = "conf/agent/dummy_root_ca.crt"
+    trust_domain = "example.org"
+    allow_unauthenticated_verifiers = true
+    allowed_foreign_jwt_claims = ["c1", "c2", "c3"]
+
+    experimental {
+        tcp_socket_port = "8082"
+    }
+}
+
+plugins {
+    plugin_type_agent "plugin_name_agent" {
+        plugin_cmd = "./pluginAgentCmd"
```

Comment:
may we provide different types of paths here? like `c:\temp\pluginAgentCmd`, `c:/temp/pluginAgentCmd`,`./pluginAgentCmd`, `.\pluginAgentCmd`?